libs/feature/record/src/lib/data-otherlinks/data-otherlinks.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="mb-4 pl-2 flex flex-wrap justify-between gap-y-6">
```

Comment:
`justify-between` will produce weird results with only two links:
![image](https://user-images.githubusercontent.com/10629150/144750233-e24c86a5-22c7-4342-9155-498e550d0df9.png)

Reply:
Thanks for testing, used `grid` now.